random-sources/photo-db/ImageToRandomNumberConvertor.py

```diff
@@ -82,11 +106,65 @@ def __split(self, img, N, W):
         new_imgs = np.array(new_imgs)
         return new_imgs
 
-    def get_avg_colours_img(self, imageFile, N, W):
+    def __get_avg_colours_img(self, imageFile, N, W):
         if( N < W ): return 0
+        
         img = cv2.imread(imageFile)
+        
         img = cv2.resize(img, (N, N)) # step 1
         imgs = self.__split(img, N, W) # step 2
         values = self.__averageOfColorsOfImages(imgs, W) # step 3
 
-        return values
\ No newline at end of file
+        return values
+        
+    
+    def __get_list_of_arrays_from_images(self, path, N, W, numOfImages):
+        x = []
+        counter = 0
+        
+        for i in sorted(os.listdir(path),key=self.__key):
+            # print(i)
```

Comment:
I think these kind of comments could be removed from all places since they don't explain anything and make the code look unclean.
Thanks for testing the code.

Reply:
Should be done